Modules/IO/XML/test/itkFancyStringTest.cxx

```diff
@@ -38,6 +38,13 @@ testFancyStringWithItkArray();
 void
 testFancyStringForStringOperations();
 
+int
```

Comment:
In commit message, `FrancyString` -> `FancyString`

Reply:
:+1: Done in 40b091c.